config.json

```diff
@@ -4300,7 +4300,7 @@
       },
       {
         "groupId": "com.squareup.okhttp3",
-        "artifactId": "okhttp",
+        "artifactId": "okhttp-android",
```

Comment:
Can we make this a new package, and move the relevant `Metadata.xml` and `Additions/*.cs` to be used in the new package?

Keep the existing `okhttp` package, and it shouldn't need `Metadata.xml` or `Additions/*.cs`.